.buildkite/lint.rayci.yml

```diff
@@ -56,14 +56,7 @@ steps:
       - ./ci/lint/check-pytest-format.sh
     depends_on: forge
 
-  - label: ":lint-roller: lint: test coverage"
+  - label: ":lint-roller: lint: test coverage "
```

Comment:
why change this file? rebase failure?

Reply:
yeah sorry, i just noticed this. working on reverting it